src/lib/components/backoffice/Sidebar/Sidebar.js

```diff
@@ -244,6 +244,10 @@ class Sidebar extends Component {
                     </Menu.Item>
                   </Menu.Menu>
                 </Menu.Item>
+                <Overridable
+                  activePath={activePath}
+                  id="Backoffice.Sidebar.MenuItem"
```

Comment:
```suggestion
                  id="Backoffice.Sidebar.ExtraMenuItem"
```
or 
```suggestion
                  id="Backoffice.Sidebar.CustomMenuItem"
```

MenuItem suggests that we overwrite the MenuItem component itself (this comes as first to my mind)